muacrypt/recommendation.py

```diff
@@ -1,34 +1,67 @@
 class Recommendation:
     """ Calculating recommendations for encryption """
 
-    def __init__(self, peerstates):
+    def __init__(self, peerstates, prefer_encrypt, reply_to_enc=False):
         self.peerstates = peerstates
+        self.prefer_encrypt = prefer_encrypt
+        self.reply_to_enc = reply_to_enc
 
     def ui_recommendation(self):
         # only consider first peer for now
-        peer = list(self.peerstates.values())[0]
-        return self._peer_recommendation(peer)
+        peer_recommendations = [
+            self._peer_recommendation(peer).ui_recommendation()
+            for peer in self.peerstates.values()]
+        precedence = ['disable', 'discourage', 'available', 'encrypt']
+        return next(rec for rec in precedence if rec in peer_recommendations)
```

Comment:
took me a little while to evaluate this line.  It's a cute one but i'd prefer it spelled out:
```
for rec in ['disable', 'discourage', 'available', 'encrypt']:
    if rec in peer_recommendations:
        return rec 
```